akka-http-jwt/src/test/resources/my-private.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Add a README.md to this directory with commands/instructions about how to generate keypair and token to make it easy for ourselves in the future